app/mailers/user_mailer.rb

```diff
@@ -14,4 +15,9 @@ def mail_reservation(event)
     mail(to: 'han@devluchte.nl', subject: "Hamers: #{event.signups.where(status: true).count} personen")
   end
 
+  def mail_new_event(user, event)
+    @event = event
+    @user = user
+    mail(to: @user.email, subject: "Nieuwe activiteit: #{event.title}")
```

Comment:
This line has inconsistent use of the class and local variables, one `@` and one without. I'm fine with both as long as they're consistent.